ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -131,12 +131,13 @@ def load_demonstration(
                     meta_data_proto = DemonstrationMetaProto()
                     meta_data_proto.ParseFromString(data[pos : pos + next_pos])
                     total_expected += meta_data_proto.number_steps
-                    pos = INITIAL_POS
-                if obs_decoded == 1:
+                    # first 32 bytes of file dedicated to metadata
+                    pos = 33
```

Comment:
I think this was better as a constant - it's a bit less "magic" that way.
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Makes sense! It just seemed weird to me to declare the constant at the beginning of the function, when it wasn't clear what it's not clear what it's going to be used for. Let me know what you think about the latest change.